docs/conf.py

```diff
@@ -222,7 +222,8 @@ def setup(app):
     "reference/experiment-config": "../reference/reference-training/experiment-config-reference.html",
     "reference/helm-config": "../reference/reference-deploy/config/helm-config-reference.html",
     "reference/historical-cluster-usage-data": "../cluster-setup-guide/historical-cluster-usage-data.html",
-    "reference/index": "../reference/python-api.html",
+    "reference/index": "../reference/python-sdk.html",
```

Comment:
This is not an issue with your pr or something you should fix in this reference, but I want to point it out since I noticed it.

It seems like `reference/index` should be an overview page with links to all the references, not a redirect to one particular reference.

Reply:
i've added this to my list for later prioritization